tests/bootstrap.php

```diff
@@ -20,6 +20,15 @@ function _manually_load_plugin() {
 	define( 'WP_CRON_CONTROL_SECRET', 'testtesttest' );
 
 	require dirname( dirname( __FILE__ ) ) . '/cron-control.php';
+
+	// Plugin loads after `wp_install()` is called, so we compensate
+	// See the `class_init()` method in Events_Store for the logic behind this
+	\Automattic\WP\Cron_Control\Events_Store::instance()->prepare_table();
+	remove_filter( 'schedule_event', '__return_false' );
+	add_filter( 'pre_option_cron', array( \Automattic\WP\Cron_Control\Events_Store::instance(), 'get_option' ) );
+	add_filter( 'pre_update_option_cron', array( \Automattic\WP\Cron_Control\Events_Store::instance(), 'update_option' ), 10, 2 );
+	add_filter( 'schedule_event', array( \Automattic\WP\Cron_Control\Events_Store::instance(), 'block_creation_if_job_exists' ) );
```

Comment:
Minor: if we add `use \Automattic\WP\Cron_Control\Events_Store` to the top of this file, we can simplify the calls to this namespace:

```
add_filter( 'pre_update_option_cron', array( Events_Store::instance(), 'update_option' ), 10, 2 );
add_filter( 'schedule_event', array( Events_Store::instance(), 'block_creation_if_job_exists' ) );
```